app/de/frosner/broccoli/nomad/models/Job.scala

```diff
@@ -0,0 +1,15 @@
+package de.frosner.broccoli.nomad.models
+
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+
+case class Job(taskGroups: Option[Seq[TaskGroup]])
```

Comment:
Please make case classes `final`, to prevent accidentally inheriting a case class (which you should never do, semantically)